src/fmpz_poly/fread_pretty.c

```diff
@@ -9,10 +9,16 @@
     (at your option) any later version.  See <https://www.gnu.org/licenses/>.
 */
 
-#include <string.h>
+#include <stdio.h>
 #include <ctype.h>
 
-#include "flint.h"
+#ifdef __GNUC__
+# define strcmp __builtin_strcmp
+# define strcpy __builtin_strcpy
+#else
+# include <string.h>
+#endif
```

Comment:
Is there any real benefit to avoiding standard includes in single files?

Reply:
No, not really. Perhaps a second faster build if done throughout the repository. Although unnecessary inclusions should be removed, doing this for the necessary ones won't really save you time.

Reply:
I thought it was going to make a difference when I started out. Do you want me to remove them?